### PR TITLE
getting files in current directory

### PR DESCRIPTION
**Pull request by Bolshartd**

### CODE REVIEW

File: files_in_root.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+def dir():
+    import os
```
Comment (vmeshche) [inline files_in_root.py]: Typically imports writes at the beginning of the file. Before any declaration

File: files_in_root.py [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+    this_os = os.getcwd()
```
Comment (vmeshche) [inline files_in_root.py]: Looks like wrong mind of the variable name. It contains current dir, not os

**Review by vmeshche — CHANGES_REQUESTED** [review]

File: files_in_root.py (hunk 1 above)
Comment (Bolshartd) [reply files_in_root.py]: Ok, i'll change their places in the new version

File: files_in_root.py (hunk 2 above)
Comment (Bolshartd) [reply files_in_root.py]: You are right, better use : this_dir. It's more understandeble. i'll ichange it in the new version.

**Pull request merged**